src/NHSD.BuyingCatalogue.Identity.Api/Services/ProfileService.cs

```diff
@@ -21,7 +21,13 @@ public sealed class ProfileService : IProfileService
         private static readonly IDictionary<string, IEnumerable<Claim>> _organisationFunctionClaims =
             new Dictionary<string, IEnumerable<Claim>>
             {
-                { "Authority", new List<Claim> { new Claim(ApplicationClaimTypes.Organisation, "Manage") } }
+                { 
+                    "Authority", new List<Claim>
+                    {
+                        new Claim(ApplicationClaimTypes.Organisation, ApplicationPermissions.Manage),
```

Comment:
Just had an idea but I wonder if it would be better to create the claims in a separate class rather than exposing all the different constants? That way the claim types and application permissions can be internal to the 'ApplicationClaims' class.

Reply:
I like the idea, but I'm inclined to make this a tech debt ticket, as it would be neater, certainly, but is a bit of work to change (mainly due to the tests) and it's currently functional and tidy.

Reply:
👍